templates/base/base.html

```diff
@@ -60,5 +60,9 @@
     </script>
     {% block scripts %}
     {% endblock scripts %}
+    {% if enable_analytics %}
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{analytics_id}}"></script>
+    <script>window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', '{{analytics_id}}');</script>
+    {% endif %}
```

Comment:
⛏️ minor suggestion for emphasising environment variable usage
```suggestion
    <script async src="https://www.googletagmanager.com/gtag/js?id={{ ANALYTICS_ID }}"></script>
    <script>window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', '{{ ANALYTICS_ID }}');</script>
    {% endif %}
```

Reply:
this needs to match the context processor. I've kept it lowercase to be consistent with how we've done the env variable

```
{"env": settings.ENV}
```